.changeset/add-chatgpt-oauth-provider.md

```
@@ -0,0 +1,5 @@
+---
+'task-master-ai': minor
+---
+
+feat: add ChatGPT OAuth provider (gpt‑5) via subscription (no API key). Wired into unified services with model `chatgpt-oauth/gpt-5` (main, fallback only), JSON object generation/validation, and optional reasoning controls (effort/summary). Docs included.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)